client/src/components/popups/TournamentPopup.tsx

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+
+interface PopupProps {
+  show: boolean;
+  onClose: () => void;
+  onConfirm: () => void;
+}
+
+const TournamentPopup: React.FC<PopupProps> = ({
+  show,
+  onClose,
+  onConfirm,
+}) => {
+  if (!show) return null; // Return null if the popup should not be shown
+
+  return (
+    <div className="fixed inset-0 flex items-center justify-center bg-black bg-opacity-70 z-50 p-4">
+      <div className="relative bg-[#2a2a72] text-white p-6 rounded-lg w-full max-w-md md:max-w-lg text-center border border-white">
+        <h2 className="text-2xl font-bold mb-4">START TOURNAMENT</h2>
```

Comment:
Can we pass children down into this component? Rename this component to Popup and then we can reuse it with whatever content we want.